app/views/studies/show.html.erb

```diff
@@ -82,6 +82,27 @@
                 </div>
             </div>
             <div class="study-main__secondary">
+              <% if %w(protocol_erb delivery).include?(@study.study_stage) && @study.user_can_manage?(current_user)%>
+                <div class="study-main__secondary__section study-main__secondary__section--admin">
+                    <h3>Manage this study:</h3>
+                    <ul>
+                      <% if @study.study_stage == 'protocol_erb' %>
+                        <li>
+                            <%= link_to "This study is in delivery",
```

Comment:
This would be better as a button that made it clear it was an action, e.g. 'Progress study to delivery' or some such, especially as there's no undo (except by an admin).